pages/home.js

```diff
@@ -13,6 +14,7 @@ export default function Home(props) {
     <div id="homeContent">
       <Heading id="my-dashboard-heading" title={t.pageHeading.title} />
       <p className="py-8">{props.content.paragraph}</p>
+      <Card t={t} />
```

Comment:
I'm conflicted over the simplicity of just passing the single prop for all text vs. specifying each one... not sure if there are other thoughts; if this is a pattern we want to continue?

Reply:
I am fine with both, I'll change it up if you prefer.